### PR TITLE
minor fix for better readability

### PR DESCRIPTION
**Pull request by dav1d337**
Fixes [#727](https://github.com/publiclab/image-sequencer/issues/727)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x ] code is in uniquely-named feature branch and has no merge conflicts
* [x ] PR is descriptively titled
* [x ] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
@dav1d337 you can replace the '#[727]' with a link to the issue and github will automatically replace it with '#727' but with a link
E.g.
https://github.com/publiclab/image-sequencer/issues/727 is a link.

If the PR body has keywords like fixes, fix, resolve, resolves followed by a link to the issue then the issue will be closed automatically once the PR is merged

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ __Moderators Group:__ The moderators group is responsible for addressing immedia
 
 We begin interactions by acknowledging that we are part of a community with complementary goals. Different views are allowed to respectfully coexist in the same space. When something's happened and someone is uncomfortable, our first choice is to work through it. Endeavor to listen and appropriately adjust your behavior if someone approaches you privately with a request that you apologize or publicly requests that you stop an ongoing presentation. If someone questions your words, actions or motives, or "calls you out", hear their feedback and respond respectfully. It’s okay to not understand why something is hurtful or causes discomfort, as long as you approach it respectfully, with empathy. Repeating hurtful behavior after it has been addressed is disrespectful and is not allowed. Doing so will result in removal and subsequent banning from in-person events and being placed into moderation in online spaces. 
 
-###The first rule of engaging with others is consent
+## The first rule of engaging with others is consent
```
Comment (HarshKhandeparkar) [inline CODE_OF_CONDUCT.md]: Could you plesse add a new `#` here? There were 3 `#`s here before. Please add a new `#`

**Comment by HarshKhandeparkar** [comment]
@dav1d337 great work but there is one error please it. Thanks!

**Comment by dav1d337** [comment]
Done (i guess :) ). Thanks for the nice task to begin with

**Comment by HarshKhandeparkar** [comment]
@dav1d337 I'm adding the label. Just make sure if there are any other places where changes are required. Thanks!

**Comment by jywarren** [comment]
Thanks, everyone!!!!

**Pull request merged**